packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1650,7 +1650,7 @@ async function handler(
 
     // Use EventManager to conditionally use all needed integrations.
 
-    const updateManager = await eventManager.reschedule(evt, originalRescheduledBooking.uid, booking?.id);
+    const updateManager = await eventManager.reschedule(evt, originalRescheduledBooking.uid);
```

Comment:
How does this change handle rescheduling for seated bookings? Or is that being handled elsewhere?

Reply:
It's being handled in the reschedule method, having that third parameter triggers that logic to run.